server/sonar-web/src/main/js/apps/organizations/components/OrganizationAdminContainer.tsx

```diff
@@ -17,19 +17,28 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-import React from 'react';
+import * as React from 'react';
 import { connect } from 'react-redux';
+import { RouterState } from 'react-router';
+import { Location } from 'history';
 import { getOrganizationByKey } from '../../../store/rootReducer';
 import handleRequiredAuthorization from '../../../app/utils/handleRequiredAuthorization';
+import { Organization } from '../../../app/types';
 
-export class OrganizationAdmin extends React.PureComponent {
-  /*:: props: {
-    children?: React.Element<*>,
-    organization: { canAdmin: boolean }
-  };
-  */
+interface StateToProps {
+  organization?: Organization;
+}
+
+interface OwnProps extends RouterState {
+  children: JSX.Element;
+}
+
+interface Props extends StateToProps {
```

Comment:
I think you should move `location` to `OwnProps` and change this line to 
```typescript
interface Props extends StateToProps, OwnProps {}
```

Reply:
Ok, I will do that, but I'll need to `Pick<OwnProps, 'children' | 'location'>`, because `OwnProps extends RouterState` that define `params`, `location` and some other stuff.